editor/src/core/model/scene-utils.ts

```diff
@@ -303,10 +303,10 @@ export function isSceneChildWidthHeightPercentage(
 }
 
 export function isDynamicSceneChildWidthHeightPercentage(
-  scene: ComponentMetadata,
+  scene: ElementInstanceMetadata,
   metadata: JSXMetadata,
 ): boolean {
-  const isDynamicScene = scene.sceneResizesContent
+  const isDynamicScene = scene.props.resizesContent ?? false
```

Comment:
I just realized `PathForResizeContent` is `resizeContent` and not `resizesContent`.

I think this "fake prop" should also use the `PathForResizeContent` const.

Reply:
Good catch. Fixing now :+1: